presto-docs/src/main/sphinx/release/release-0.247.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.247
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug for reporting output data sizes for optimized repartitioning.
+* Fix accounting for revocable memory that could cause some queries not to spill when they should.
+* Fixed a race condition in enum key lookup which caused queries using the `ENUM_KEY` UDF to crash occasionally with an internal error #15607.
+* Add function ``array_intersect`` that takes an array of array as input.
+* Add support for temporary (session-scoped) functions.
+* Adding retry behavior for verifier requests in case of throttling error.
```

Comment:
This needs to move to a new section `Verifier changes` - https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-sections

Reply:
`Add retries for verifier queries that fail due to HTTP Error 429 (Too many requests)`